.env.dist

```diff
@@ -0,0 +1 @@
+COMPOSE_FILE=.docker/docker-compose.yml
```

Comment:
The dot is a typo or is it usefull?

Reply:
The `Dockerfile` and `docker-compose.yml` file are both stored in a `.docker` folder at the root of the project. This is a convention we use to distinct everything that is not directly part of a project (like .circleci for example).

Reply:
It's the name of the directory `.docker`

Reply:
> This is a convention we use to distinct everything that is not directly part of a project

TBH, this was named like this on other JS projects, but I'm not fan of it as it implies that it's an optional stuff :/ Whereas the app is meant to run on docker so I prefer to use the `docker` name.